### DISCUSSION TITLE
[Help]: Vue/Nuxt implementation with navigation and paging

### DISCUSSION THREAD

**Discussion by codeflorist** [davidjerleke/embla-carousel#1055] [Help] [answered]
### Summary

Does anyone have an example implementation for Vue (preferably Nuxt)?

I was looking for an alternative to Swiper, and Embla looks cool, but it seems all very confusing and complex to implement simple paging and navigation, which other carousel-solutions have on board. It seems one has to program the complete handling of these per hand using the emblaApi.

Afaiks the documentation regarding this is all hidden inside the examples of the website, which are only React and vanilla js, but not vue.

I could not find examples for complete vue components, that include paging and navigation. Does anyone happen to have such components somewhere?

### If applicable, which variants of Embla Carousel are relevant to this conversation?

- [ ] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [X] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by davidjerleke** [comment, marked as answer]
@codeflorist my spare time is very limited so that's the reason Vue, Svelte and Solid examples are missing in the docs. Here's a sandbox that demonstrates how you set up navigation buttons using Vue:

- https://github.com/davidjerleke/embla-carousel/issues/383#issuecomment-1278993224

**Reply by codeflorist** [reply]
Many thanks!
